lib/node-utils/index.ts

```diff
@@ -510,3 +532,95 @@ export function hasImportMatch(
 
   return importNode.local.name === identifierName;
 }
+
+export function getStatementCallExpression(
+  statement: TSESTree.Statement
+): TSESTree.CallExpression | undefined {
+  if (
+    isExpressionStatement(statement) &&
+    isCallExpression(statement.expression)
+  ) {
+    return statement.expression;
+  }
+
+  if (isReturnStatement(statement) && isCallExpression(statement.argument)) {
+    return statement.argument;
+  }
+
+  if (isVariableDeclaration(statement)) {
+    for (const declaration of statement.declarations) {
+      if (isCallExpression(declaration.init)) {
+        return declaration.init;
+      }
+    }
+  }
+  return undefined;
+}
+
+/**
+ * Determines whether a given function node is considered as empty function or not.
+ *
+ * A function is considered empty if its body is empty.
+ *
+ * Note that comments don't affect the check.
+ *
+ * If node given is not a function, `false` will be returned.
+ */
+export function isEmptyFunction(node: TSESTree.Node): boolean | undefined {
+  if (ASTUtils.isFunction(node) && isBlockStatement(node.body)) {
+    return node.body.body.length === 0;
+  }
+
+  return false;
```

Comment:
```suggestion
  return ASTUtils.isFunction(node) && isBlockStatement(node.body) && node.body.body.length === 0
```

Minor, but I think the if can be simplified just to

Reply:
Not sure if more readable tho 😅